docs/content/direct/core-chart-argocd.md

```diff
@@ -0,0 +1,51 @@
+# Using Argo CD with KubeStellar Core chart
+
+This documents explains how to use the KubeStellar core Helm chart to:
+
+- deploy Argo CD in the KubeFlex hosting cluster;
+- register every WDS as a target cluster in Argo CD; and
+- create Argo CD applications as specified by the chart values.
+
+## Pre-requisites
+
+The prerequisites are the same as [installing KubeStellar using the Core chart](core-chart.md#pre-requisites).
+
+The settings described in this document are an extesnion of the KubeStellar Core chart settings described [here](core-chart.md#kubestellar-core-chart-values).
+
+## Installing Argo CD using KubeStellar Core chart
+
+To enable the installation of Argo CD by the KubeStellar Core chart, use the flag `--set "argo-cd.install=true"`. Besides deploying an instance of Argo CD, KubeStellar Core chart will take care of registering all the WDSes installed by the chart as Argo CD target clusters.
+
+When deploying in an **OpenShift** cluster, add the flag `--set "argo-cd.openshift.enabled=true"`.
+
+When deploying in a **Kubernetes** cluster, use the flag `--set "argo-cd.global.domain=<url>"` to provide the URL for the **nginx** ingress, default `argocd.localtest.me`.

```

Comment:
s/default/which defaults to/

I would add a remark at the end that this default works for clusters created by `kind` and `k3d` but not in general.